src/main/resources/assets/systeams/lang/ko_kr.json

```diff
@@ -0,0 +1,75 @@
+{
+  "item.systeams.steam_bucket": "증기 양동이",
+  "fluid_type.systeams.steam": "증기",
+  "item.systeams.boiler_pipe": "증기 파이프",
+  "item.systeams.boiler_pipe.keyword": "증기 발전기 보일러 코일",
+  "item.thermal.rf_coil.keyword": "증기 발전기 보일러",
+  "item.systeams.boiler_pipe.desc": "Shift-사용으로 발전기를 보일러로 변환합니다. 주요 부품을 회수합니다",
+  "item.thermal.rf_coil.desc": "Shift-사용으로 보일러를 발전기로 변환합니다. 증기 파이프를 회수합니다",
+  "itemGroup.systeams": "Systeams",
+
+  "info.systeams.steam": "증기",
+  "info.systeams.steam_prod": "증기 생산",
+  "info.systeams.water_cons": "물 소모",
+
+  "info.systeams.air": "공기",
+  "info.systeams.air_pressure": "압력",
+  "info.systeams.air_volume": "공기",
+  "info.systeams.air_base_volume": "기본 용량",
+  "info.systeams.air_max_usage": "최대 사용량",
+  "info.systeams.air_volume_unit": "mL",
+  "info.systeams.air_per_tick": "최대 공기 사용량",
+  "info.systeams.air_per_tick_unit": "mL/틱",
+
+  "block.pneumaticcraft.advanced_pressure_tube.desc": "Shift-사용으로 발전기를 보일러로 변환합니다. 증기 파이프를 회수합니다",
+
+  "block.systeams.steam_dynamo": "증기 발전기",
+  "info.systeams.steam_dynamo.0": "증기를 동력으로 전환합니다.",
+  "info.systeams.steam_dynamo.1": "여기 점점 찜찜해지는데?",

```

Comment:
If this (or any other) joke/pun doesn't translate well into Korean feel free to replace it with a different (but still related to steam power generation) one that is funny in Korean. Or leave it as it is and make a comment on the PR if you can't come up with another (or if it is already funny enough in Korean).

Reply:
yeah, that's enough for now.